packages/node/src/integrations/utils/http.ts

```diff
@@ -168,6 +168,16 @@ export function normalizeRequestArgs(
     requestOptions = urlToOptions(requestArgs[0]);
   } else {
     requestOptions = requestArgs[0];
+
+    if (!requestOptions.pathname || !requestOptions.search) {
+      const parsed = new URL(
```

Comment:
Two things:

1. Let's try-catch this to ensure this does not fail if we cannot generate an URL there, just to be on the safe side
2. Let's instead set the parsed values only if they aren't set. 

So something like this:

```js
try {
  const parsed = new URL(...);
  requestOptions = { 
    pathname: parsed.pathname, 
    search: parsed.search, 
    hash: parsed.hash,
    ...requestOptions
  };
} catch (e) {
  // ignore errors here
}
```